app/views/candidate_interface/references/review/show.html.erb

```diff
@@ -1,4 +1,17 @@
-<% content_for :title, t('page_titles.references') %>
+<% if @too_many_references %>
+  <div class="govuk-error-summary" tabindex="-1" role="alert" data-module="govuk-error-summary" aria-labelledby="error-summary-title">
+    <h2 id="error-summary-title" class="govuk-error-summary__title">
+      There is a problem
+    </h2>
+    <div class="govuk-error-summary__body">
+      <ul class="govuk-list govuk-error-summary__list">
+        <li><%= link_to t('application_form.references.review.more_than_two'), '#references_given' %></li>
+      </ul>
+    </div>
+  </div>
+<% end %>
+
+<% content_for :title, title_with_error_prefix(t('page_titles.references'), @too_many_references) %>
 <% content_for :before_content, govuk_back_link_to(candidate_interface_application_form_path, 'Back to application') %>
 
```

Comment:
I think lines 1-12 should be here. It's normally: 

backlink

errors

title

Reply:
updated!